semgrep/semgrep/ignores.py

```diff
@@ -41,14 +41,13 @@ def process_ignores(
     :param strict: The value of the --strict flag (affects error return)
     :param disable_nosem: The value of the --disable-nosem flag
     :return:
-    - RuleMatchMap with nosem findings removed if disable_nosem is true
+    - RuleMatchMap: dict from rule to list of findings. Findings have is_ignored
+        set to true if there was matching nosem comment found for it.
+        Contains only findings without nosem if disable_nosem set to False
```

Comment:
So many negatives...maybe "contains findings with nosem unless disable_nosem is passed"?

Reply:
I'll follow up in with a refactor from disable_nosem to filter_nosem or something to make it more clear